CHANGES.rst

```diff
@@ -5,14 +5,15 @@
 5.2.1 (unreleased)
 ==================
 
-- Nothing changed yet.
-
+- Fixed: When opening FileStorages in read-only mode, non-existent
+  files were silently created.  Creating a file-storage against a
```

Comment:
"Creating a read-only FileStorage..."

Leaving out the 'read-only' makes that sentence sound a little scary.